tool/tctl/common/collection.go

```diff
@@ -1733,3 +1735,60 @@ func (c *spiffeFederationCollection) writeText(w io.Writer, verbose bool) error
 	_, err := t.AsBuffer().WriteTo(w)
 	return trace.Wrap(err)
 }
+
+type staticHostUserCollection struct {
+	items []*userprovisioningpb.StaticHostUser
+}
+
+func (c *staticHostUserCollection) resources() []types.Resource {
+	r := make([]types.Resource, 0, len(c.items))
+	for _, resource := range c.items {
+		r = append(r, types.Resource153ToLegacy(resource))
+	}
+	return r
+}
+
+func (c *staticHostUserCollection) writeText(w io.Writer, verbose bool) error {
+	var rows [][]string
+	for _, item := range c.items {
+
+		for _, matcher := range item.Spec.Matchers {
+			labelStrings := make([]string, 0, len(matcher.NodeLabels))
+			for _, label := range matcher.NodeLabels {
+				labelStrings = append(labelStrings, fmt.Sprintf("%s=[%s]", label.Name, printSortedStringSlice(label.Values)))
+			}
+			var uid string
+			if matcher.Uid != 0 {
+				uid = strconv.Itoa(int(matcher.Uid))
+			}
+			var gid string
+			if matcher.Gid != 0 {
+				gid = strconv.Itoa(int(matcher.Gid))
+			}
+			rows = append(rows, []string{
+				item.GetMetadata().Name,
+				printSortedStringSlice(labelStrings),
+				matcher.NodeLabelsExpression,
+				printSortedStringSlice(matcher.Groups),
+				uid,
+				gid,
+			})
+		}
```

Comment:
could I suggest this patch to make use of the common label formatting:

```diff
diff --git a/tool/tctl/common/collection.go b/tool/tctl/common/collection.go
index 9b85969a8b..f8e18d2444 100644
--- a/tool/tctl/common/collection.go
+++ b/tool/tctl/common/collection.go
@@ -43,6 +43,7 @@ import (
 	"github.com/gravitational/teleport/api/types/accesslist"
 	"github.com/gravitational/teleport/api/types/discoveryconfig"
 	"github.com/gravitational/teleport/api/types/externalauditstorage"
+	"github.com/gravitational/teleport/api/types/label"
 	"github.com/gravitational/teleport/api/types/secreports"
 	apiutils "github.com/gravitational/teleport/api/utils"
 	"github.com/gravitational/teleport/lib/asciitable"
@@ -1753,9 +1754,10 @@ func (c *staticHostUserCollection) writeText(w io.Writer, verbose bool) error {
 	for _, item := range c.items {
 
 		for _, matcher := range item.Spec.Matchers {
-			labelStrings := make([]string, 0, len(matcher.NodeLabels))
-			for _, label := range matcher.NodeLabels {
-				labelStrings = append(labelStrings, fmt.Sprintf("%s=[%s]", label.Name, printSortedStringSlice(label.Values)))
+			labelMap := label.ToMap(matcher.NodeLabels)
+			labelStringMap := make(map[string]string)
+			for k, vals := range labelMap {
+				labelStringMap[k] = fmt.Sprintf("[%s]", printSortedStringSlice(vals))
 			}
 			var uid string
 			if matcher.Uid != 0 {
@@ -1767,7 +1769,7 @@ func (c *staticHostUserCollection) writeText(w io.Writer, verbose bool) error {
 			}
 			rows = append(rows, []string{
 				item.GetMetadata().Name,
-				printSortedStringSlice(labelStrings),
+				common.FormatLabels(labelStringMap, verbose),
 				matcher.NodeLabelsExpression,
 				printSortedStringSlice(matcher.Groups),
 				uid,

```

I would make a github suggestion in-line but the diff it renders is very unhelpful to see what changes I'm suggesting